packages/frontend/src/components/Trade/Long/index.tsx

```diff
@@ -560,6 +560,7 @@ const OpenLong: React.FC<BuyProps> = ({ activeStep = 0 }) => {
                   <Metric
                     label="Price Impact"
                     value={formatNumber(priceImpact) + '%'}
+                    textColor={priceImpactColor}
```

Comment:
imo price impact should only have a color if its too high and then it should be red 

if its not too high can just leave it gray - having it green has confused people, because price impact always means a loss